packages/showcases/svelte/stories/components/navigation/VtmnBreadcrumbItem/VtmnBreadcrumbItem.stories.svelte

```diff
@@ -0,0 +1,47 @@
+<script>
```

Comment:
Not necessary to have a `VtmnBreadcrumbItem` story.
The `VtmnBreadcrumb` story is enough :)

Reply:
Done

Reply:
@fgarci22 thanks!